CMakeLists.txt

```diff
@@ -134,69 +156,50 @@ else()
 endif()
 
 # Compiler / Linker flags
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fno-builtin")
+jerry_add_compile_flags(-fno-builtin)
 if(("${PLATFORM}" STREQUAL "DARWIN"))
-  set(LINKER_FLAGS_COMMON "-lSystem")
+  jerry_add_link_flags(-lSystem)
```

Comment:
This rewrite and the rewrite 2 lines below are not equivalent to their original versions. However, I suspect that the old versions (which don't append to LINKER_FLAGS_COMMON but set it) are incorrect (as LINKER_FLAGS_COMMON might have already been set for LTO).